ProcessLib/HT/HTProcess.cpp

```diff
@@ -46,7 +48,10 @@ void HTProcess::initializeConcreteProcess(
     MeshLib::Mesh const& mesh,
     unsigned const integration_order)
 {
-    ProcessLib::ProcessVariable const& pv = getProcessVariables()[0];
+    // For the staggered scheme, both processes are assumed to use the same
+    // element order.
+    const int process_id = 0;
+    ProcessLib::ProcessVariable const& pv = getProcessVariables(process_id)[0];
```

Comment:
How is the above comment related to the `process_id`? I'd expect the general comments on implementation to be in the header file and in doxygen comments....

Reply:
Added more a detailed description.